docs/beforeremove.md

```diff
@@ -1,6 +1,8 @@
 @typedef {function} can-component/beforeremove beforeremove
 @parent can-component.deprecated
 
+@deprecated Use [can-component/connectedCallback] instead.
```

Comment:
It’d be good to link to the part of the migration guide where this is covered: “inserted/removed event” in https://canjs.com/doc/migrate-4.html

That section doesn’t have an ID you can easily link to, but I think it’d be good if it did. Right now you can increase the outline for that page to make it 1) display more in the TOC and 2) generate the required IDs. I think something like `@outline 2` on that page would do the trick, but you’ll need to check (and open a PR).

Also, I think you can say this was deprecated in 4.0

Reply:
Done!